backend/grant/proposal/models.py

```diff
@@ -706,6 +725,13 @@ def get_date_published(self, obj):
     def get_is_version_two(self, obj):
         return True if obj.version == '2' else False
 
+    def get_is_subscribed(self, obj):
+        if current_user.is_authenticated and not current_user.banned:
+            return obj.is_subscribed(current_user)
+        else:
+            return False
+
```

Comment:
Are you intending to get a boolean response here? If so, can be simplified to one-liner: 

`current_user.is_authenticated and not current_user.banned and obj.is_subscribed(current_user)`.